com.emqx.MQTTX.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: com.emqx.MQTTX
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+# Electron doesn't use a traditional locale format
+# so separate-locales is useless.
+separate-locales: false
+command: mqttx.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=~/.config/MQTTX
+modules:
+  - name: MQTTX
+    buildsystem: simple
+    build-commands:
+      - chmod +x MQTTX.AppImage
+      - ./MQTTX.AppImage --appimage-extract
+      - cp -r squashfs-root/ /app/mqttx
+      - install -D mqttx.sh /app/bin/mqttx.sh
+      - install -D squashfs-root/mqttx.png /app/share/icons/hicolor/512x512/apps/com.emqx.MQTTX.png
+      - install -D com.emqx.MQTTX.desktop /app/share/applications/com.emqx.MQTTX.desktop
```

Comment:
doesn't the appimage come with its .desktop file?

Reply:
Sorry! I don't seem to know too well. :)

Reply:
Looks like it doesn't.